mythical.py

```diff
@@ -10,7 +10,7 @@ def keliauti(self, vietove):
         self.jegos -= 10
 
     def dainuoti(self):
-        print("lia lia lia, tra lia lia")
+        print("Unt Sartų unt Sartų lakia roges ratu")
```

Comment:
nekeisk pačio šito pagrindinio failo :) Tik savo :) Jei nori pakeisti dainavvimą, galima pridėti į savo klasę Asilas šitą metodą, ir jame įrašyti šitą tekstą

Reply:
visiems